courses/hometask6/task2/client/client.go

```diff
@@ -0,0 +1,54 @@
+// This is a simple client package
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+)
+
+func main() {
+
+	for {
+		// We read from the console
+		fmt.Println("Enter the message:")
+		reader := bufio.NewReader(os.Stdin)
+		text, _ := reader.ReadString('\n')
+
+		// If exit - the program exits
+		if strings.TrimSpace(text) == "exit" {
+			os.Exit(0)
+		}
+
+		// We dial to the server
+		conn, ok := net.Dial("tcp", "127.0.0.1:8083")
```

Comment:
It's possible to open a connection and wait for incoming messages from STDIN, while this approach is OK too.